packages/okta-react-native/index.js

```diff
@@ -0,0 +1,94 @@
+import { NativeModules, Platform } from 'react-native';
+import {
+	assertIssuer,
+  assertClientId,
+	assertRedirectUri
+} from '@okta/configuration-validation';
+import jwt from 'jwt-lite';
+
+export const createConfig = async (
+  clientId,
+  redirectUri, 
+  endSessionRedirectUri, 
+  discoveryUri,
+  scopes) => {
```

Comment:
That is a very long function signature.  Have you considered using object destructuring?  
```
export const createConfig = async ({ 
  clientId, 
  redirectUri, 
  endSessionRedirectUri, 
  discoveryUri,
  scopes
}) => { 
```

That would require changing how this function is called (you pass an object of all the params rather than just the params) but has 2 key advantages:
* The order of the params no longer matters, as they are identified by key name rather than position
* Each argument is essentially named by the key, making reading the code more clear, particularly with many function arguments.
e.g. `createConfig({ clientId: 'DFG352SF2SR', redirectUri: 'http://myapp.com/', // etc })`

Object property name shorthand means that when variable names match up to the property names this can be less tedious:

```
const clientId = '123456';
const redirectUri = 'http://myapp.com/';
createConfig({ clientId, redirectUri, //etc}); 
// same as
// createConfig({ clientId: clientId, redirectUri: redirectUri //etc })
```

Object destructuring for function params may not always be the best choice, but I find I always want it when I have 3 or more params.

Reply:
that does sound good to me!